wallet/src/background/Permissions.ts

```diff
@@ -17,12 +17,14 @@ import type {
 function openPermissionWindow(permissionID: string) {
     return new Window(
         Browser.runtime.getURL('ui.html') +
-            `#/connect/${encodeURIComponent(permissionID)}`
+            `#/dapp/connect/${encodeURIComponent(permissionID)}`
     );
 }
 
 const PERMISSIONS_STORAGE_KEY = 'permissions';
 
+// Browser.storage.local.clear();
```

Comment:
do we need this?

Reply:
Nope I’ll remove, was just debugging some stuff

Reply:
Nope I’ll remove, was just debugging some stuff